classes/submission/Collector.php

```diff
@@ -57,9 +57,7 @@ public function getQueryBuilder(): Builder
             $q->whereIn('s.submission_id', function ($query) {
                 $query->select('issue_p.submission_id')
                     ->from('publications AS issue_p')
-                    ->join('publication_settings as issue_ps', 'issue_p.publication_id', '=', 'issue_ps.publication_id')
-                    ->where('issue_ps.setting_name', '=', 'issueId')
-                    ->whereIn('issue_ps.setting_value', array_map('strval', $this->issueIds));
+                    ->whereIn('issue_p.issue_id', $this->issueIds);
             });
```

Comment:
The parent class has a join (i.e. `publications AS po`) that you might use, and it will greatly simplify these lines of code.